Source/Core/VideoCommon/OnScreenDisplay.cpp

```diff
@@ -111,11 +111,18 @@ static float DrawMessage(int index, Message& msg, const ImVec2& position, int ti
       {
         ImGui::Image(msg.texture.get(), ImVec2(static_cast<float>(msg.icon->width),
                                                static_cast<float>(msg.icon->height)));
+        ImVec2 cursor_position = ImGui::GetCursorPos();
```

Comment:
I think this might be similar if you want to try it:

```
if (msg.text.size() > 0)
{
  ImGui::SameLine();
  ImGui::Dummy(ImVec2{WINDOW_PADDING * ImGui::GetIO().DisplayFramebufferScale.x, 0});
}
```

Reply:
Duh, I forgot I did this twice, thanks.

Reply:
Also, ImGui seems pretty good at padding on its own, I was the one who was neglecting it by moving these things manually. Not sure Dummy is all that necessary.